Ruby/wc/file_content.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# class to provide file utility information.
+class FileContent
+  def initialize(file_content)
+    @file_content = file_content
+  end
+
+  def size
+    @file_content.bytesize
+  end
+
+  def lines_count
+    lines = @file_content.lines
+    lines.count
+  end
+
+  def word_count
+    @file_content.split(/\s+/).length
```

Comment:
NIT: In the event we're just working with strings -  may be the regexp can be a default value to this function that way the caller can supply a different word separator if they want - I'm thinking different languages - but that's just a personal pref at this point. 🤷🏼‍♂️

Reply:
I didn't test with other types of input. but that's a good point.

I will explore more here.